server/events/project_command_builder.go

```diff
@@ -402,6 +433,10 @@ func (p *DefaultProjectCommandBuilder) buildAllCommandsByCfg(ctx *command.Contex
 			return nil, errors.Wrapf(err, "parsing %s", repoCfgFile)
 		}
 		ctx.Log.Info("successfully parsed %s file", repoCfgFile)
+		// This above condition to set this may not have been reached
```

Comment:
This comment isn't clear on what it is trying to convey.

Reply:
Yeah, that read better in my head. Fixed!